local Net = require(script.Parent.Parent.clientNet)

local DayNightRenderer = require(script.DayNightRenderer)
local EventRenderer = require(script.EventRenderer)
local BGMService = require(script.Parent.SoundHandler.BGMService)
local AmbientService = require(script.Parent.SoundHandler.AmbientService)

Net.DayNight.On(function(Data)
    print("DayNight Data", Data)
    BGMService.updateBGM(Data)
    AmbientService.updateAmbient(Data)
    DayNightRenderer.applyDayNight(Data)
end)
Net.Event.On(function(Data)
    print("Events Data", Data)
    BGMService.updateBGM(Data)
    AmbientService.updateAmbient(Data)
    EventRenderer.applyEvents(Data)
end)
return {} 